### DISCUSSION TITLE
Compile flycast on rpi

### DISCUSSION THREAD

**Discussion by mixerreq** [flyinghead/flycast#811] [Q&A]
How i can compile flycast on debian 10 

platform: rpi3b+
arch: armv7l

**Comment by vkedwardli** [comment]
I think I have successfully compile it on pi4 before, but pi3b+ performance should be not enough for emulating Dreamcast?

Which cmake command did you try and what is the error are you facing?

**Reply by mixerreq** [reply]
Thanks for reply. I seen compiled version of flycast on batocera 35 for rpi3. If DC game have 60 fps i set 1 frameskip and now it's playable with widescreen hack. If i only must compile on 64bit Debian 11 i can do it.
